core/server/data/sitemap/manager.js

```diff
@@ -72,138 +72,6 @@ _.extend(SiteMapManager.prototype, {
-        if (!this.initialized) {
-            return;
-        }
-
-        var pageData = page.toJSON(),
-            wasPublished = page.updated('status') === 'published',
-            isPublished = pageData.status === 'published';
-
-        // Published status hasn't changed and it's published
-        if (isPublished === wasPublished && isPublished) {
-            this.pages.updateUrl(pageData);
-        } else if (!isPublished && wasPublished) {
-            // Handle page going from published to draft
-            this.pageDeleted(page);
-        } else if (isPublished && !wasPublished) {
-            // ... and draft to published
-            this.pageAdded(page);
-        }
-    },
-
-    pageDeleted: function (page) {
-        if (!this.initialized) {
-            return;
-        }
-
-        this.pages.removeUrl(page.toJSON());
-    },
-
-    postAdded: function (post) {
-        if (!this.initialized) {
-            return;
-        }
-
-        if (post.get('status') !== 'published') {
-            return;
-        }
-
-        this.posts.addUrl(post.toJSON());
-    },
-
-    postEdited: function (post) {
-        if (!this.initialized) {
-            return;
-        }
-
-        var postData = post.toJSON(),
-            wasPublished = post.updated('status') === 'published',
-            isPublished = postData.status === 'published';
-
-        // Published status hasn't changed and it's published
-        if (isPublished === wasPublished && isPublished) {
-            this.posts.updateUrl(postData);
-        } else if (!isPublished && wasPublished) {
-            // Handle post going from published to draft
-            this.postDeleted(post);
-        } else if (isPublished && !wasPublished) {
-            // ... and draft to published
-            this.postAdded(post);
-        }
-    },
-
-    postDeleted: function (post) {
-        if (!this.initialized) {
-            return;
-        }
-
-        this.posts.removeUrl(post.toJSON());
-    },
-
-    userAdded: function (user) {
-        if (!this.initialized) {
-            return;
-        }
-
-        this.authors.addUrl(user.toJSON());
-    },
-
-    userEdited: function (user) {
-        if (!this.initialized) {
-            return;
-        }
-
-        var userData = user.toJSON();
-
-        this.authors.updateUrl(userData);
-    },
-
-    userDeleted: function (user) {
-        if (!this.initialized) {
-            return;
-        }
-
-        this.authors.removeUrl(user.toJSON());
-    },
-
-    tagAdded: function (tag) {
-        if (!this.initialized) {
-            return;
-        }
-
-        this.tags.addUrl(tag.toJSON());
-    },
-
-    tagEdited: function (tag) {
-        if (!this.initialized) {
-            return;
-        }
-
-        this.tags.updateUrl(tag.toJSON());
-    },
-
-    tagDeleted: function (tag) {
-        if (!this.initialized) {
-            return;
-        }
-
-        this.tags.removeUrl(tag.toJSON());
-    },
```

Comment:
I think it's great to move all this deleted code out of here and closer to where it's emitted.